src/webextension/manage/components/send-feedback.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Localized } from "fluent-react";
+import React from "react";
+
+import Button from "../../widgets/button";
+
+export default function SendFeedback() {
+  return (
+    <Localized id="toolbar-send-feedback">
+      <Button onClick={() => { alert("Sorry, no feedback yet"); }}>
```

Comment:
@jimporter while you're in here, can you please swap out the alert and open this URL (however is best, I'm not sure here) provided by @sandysage: https://qsurvey.mozilla.com/s3/Lockbox-Input